backend/src/test/java/ai/verta/modeldb/RepositoryTest.java

```diff
@@ -166,7 +178,8 @@ public void createDeleteRepositoryTest() {
 
     long id = createRepository(versioningServiceBlockingStub, NAME);
 
-    DeleteRepositoryRequest deleteRepository =
+    final DeleteRepositoryRequest deleteRepository;
+    deleteRepository =
         DeleteRepositoryRequest.newBuilder()
             .setRepositoryId(RepositoryIdentification.newBuilder().setRepoId(id))
             .build();
```

Comment:
Initialize this as inline like 
```suggestion
    final DeleteRepositoryRequest deleteRepository =
        DeleteRepositoryRequest.newBuilder()
            .setRepositoryId(RepositoryIdentification.newBuilder().setRepoId(id))
            .build();
```